lesson2/homework_3(x^y).py

```diff
@@ -0,0 +1,9 @@
+if __name__ == '__main__':
+    x = 25
+    n = 5
+    i = 1
+    num = 1
+    for i in range(n):
+        num = num*x
+        i += 1
```

Comment:
Why you need  i += 1? Python automatically updates i on every iteration.

Reply:
Done